client/settings/settings-manager/index.js

```diff
@@ -24,41 +74,16 @@ const SettingsManager = ( {
 
 	return (
 		<div className="settings-manager">
-			<SettingsSection
-				title={ __(
-					'Payments accepted on checkout',
-					'woocommerce-payments'
-				) }
-				description={ __(
-					'Add and edit payments available to customers at checkout. Drag & drop to reorder.',
-					'woocommerce-payments'
-				) }
-			>
+			<SettingsSection Description={ PaymentMethodsDescription }>
 				<PaymentMethods
 					enabledMethodIds={ enabledPaymentMethodIds }
 					onEnabledMethodIdsChange={ setEnabledPaymentMethodIds }
 				/>
 			</SettingsSection>
-			<SettingsSection
-				title={ __(
-					'Digital wallets & express payment methods',
-					'woocommerce-payments'
-				) }
-				description={ __(
-					// eslint-disable-next-line max-len
-					'Let customers use express payment methods and digital wallets like Apple Pay and Google Pay for fast & easy checkouts.',
-					'woocommerce-payments'
-				) }
-			>
+			<SettingsSection Description={ DigitalWalletsDescription }>
 				<DigitalWallets />
 			</SettingsSection>
-			<SettingsSection
-				title={ __( 'Settings', 'woocommerce-payments' ) }
-				description={ __(
-					"Change WooCommerce Payments settings and update your store's configuration to ensure smooth transactions.",
-					'woocommerce-payments'
-				) }
-			>
+			<SettingsSection Description={ GeneralSettingsDescription }>
```

Comment:
This might be a convention I am unaware of but shouldn't the `Description` prop be actually named `description`, and then destructured in each place where it is used: `{ description: Description }`?

Reply:
Good observation! I capitalized it to make it clearer that the prop being passed is a component - I'm not sure if it's a convention in other parts of this app, but I've been using this convention in other apps :) 

If it were lowercase, I would normally expect to pass it a node (i.e.: `description={ <GeneralSettingsDescription /> }`). But by capitalizing it, I try to infer that it should be a function/a component.

As for reasoning, if I were to pass JSX, there would be a couple of issues with re-rendering.
Since React components generally re-render when the props change, providing a function for a prop would help ensuring that the prop's reference is more "stable".
i.e.:
```jsx
const SettingsSectionWithComponent = ({ Description }) => <Description />;
const SettingsSectionWithNode = ({ description }) => description;
const DescriptionComponent = <div>Hullo!</div>;

const NodeConsumer = (props) => (
  // do stuff with the other props
  return <SettingsSectionWithNode description={ <DescriptionComponent /> } />
};
// 👆 this would cause more re-renders than this other one 👇
// because the `description` reference changes each time the props passed to `NodeConsumer` also change
// whereas the `Description` reference below is more "stable", it doesn't change at each re-render
const ComponentConsumer = (props) => (
  // do stuff with the other props
  return <SettingsSectionWithComponent Description={ DescriptionComponent } />
};

// but you still need to be careful not to do this,
// in this case the `Description` reference changes at each re-render,
// it doesn't have a "stable" reference
const BadComponentConsumer = (props) => (
  // do stuff with the other props
  return <SettingsSectionWithComponent Description={ () => <DescriptionComponent /> } />
};
```

Sorry if I over-explained :D

Reply:
Great points, all very valid, and thanks so much for writing the fantastic explanation! These are all great things to consider. TIL.

My only worry was that we might be doing something against a convention that says to always use lowercase props or something. The reasoning is very solid though.

Reply:
You're right - I created a P2 post on the WCShip team and x-posted to codep2 to gather people's feedback: paJDYF-1wZ-p2